pfcpiface/up4.go

```diff
@@ -128,7 +130,40 @@ func (m meter) String() string {
 }
 
 func (up4 *UP4) addSliceInfo(sliceInfo *SliceInfo) error {
-	log.Errorln("Slice Info not supported in P4")
+	//FIXME: UP4 currently supports a single slice meter rate common between UL and DL traffic. For this reason, we
+	//  configure the meter with the largest slice MBR between UL and DL.
+	if !up4.isConnected(nil) {
```

Comment:
This makes `addSliceInfo` fail if fastpath is not connected, without trying to do "lazy" re-connect. I suggest using `tryConnect()`, which is thread-safe, and will re-connect if not yet connected.

Reply:
see `sendMsgToUPF` for reference